catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -285,4 +305,61 @@ public boolean matches(Object o) {
             return test.test((QueryRequest) o);
         }
     }
+
+    private void mockMultipleQueries(int pageSize, int resourceLimit) throws Exception {
+
+        mockResourceResponse();
+        QueryResponse queryResponseMock = mock(QueryResponse.class);
+
+        // have as many results as the resource limit will allow
```

Comment:
✏️ Perhaps clarify this comment a bit to something more like "Populate list of mock results, sized at max resource limit" to help readers know the purpose of the lists.